packages/python-packages/apiview-stub-generator/tests/azure_tests/azure_test.py

```diff
@@ -75,8 +78,25 @@ def _get_src(src_dir, subdirectory, package_name, version):
     
     _copy_directory_from_github(src_dir, repo_url, directory, tag)
     print(f"Directory copied to: {src_dir}")
-    return src_dir
+    # Check if apiview-properties.json exists in the source directory
+    mapping_file_src = os.path.join(src_dir, "apiview-properties.json")
+    mapping_file = mapping_file_src if os.path.exists(mapping_file_src) else None
+    return src_dir, mapping_file
 
+def _get_mapping_file(dest_dir, subdirectory, package_name, version):
```

Comment:
Is this relevant to the PR, or something extra?

Reply:
Something extra, but will need to be added soon anyway b/c we're going to pass the mapping file in the CI for auto generated APIViews.